cpp/src/sort/faster_sort_column_impl.cuh

```diff
@@ -111,12 +122,45 @@ struct faster_sorted_order_fn {
 
   template <typename T>
   void operator()(mutable_column_view& input,
-                  mutable_column_view& indices,
+                  mutable_column_view& indices,  // no longer preload this
                   bool ascending,
```

Comment:
We could forward the enum class all the way through to this call site, so that we don't have a bool here.

Reply:
I suppose. I'm not convinced that is better internally here.